cogs/yourlsClient/helpers.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+import discord
+from redbot.core.utils import AsyncIter, chat_formatting
+
+
+async def createSimplePages(
+    items: List[str], embedTitle: str = "", embedAuthor: discord.Member = None
+):
+    """Create embed pages for the redbot menu coroutine.
+
+    It features the following:
+    - Each entry is numbered.
+    - The total number of items is visible.
+    - The total number of pages is visible.
+
+    Parameters
+    ----------
+    items: [ str ]
+        A list of strings you wish to display to the user.
+    embedTitle: Optional[ str ]
+        The title for all the embed pages
+    embedAuthor : Optional[ discord.Member ]
```

Comment:
```suggestion
    embedAuthor: Optional[ discord.Member ]
```
No space before a colon.